Resources/views/Menu/sonata_menu.html.twig

```diff
@@ -41,9 +41,16 @@
         <a href="#">
             {% set translation_domain = item.extra('label_catalogue') %}
             {% set icon = item.extra('icon')|default('') %}
+            {% set spanElement_parent = parent() %}
```

Comment:
Oh right, I didn't see that in the diff. It makes sense, you can keep it like this. :+1: